integration/tests/access/access_api_test.go

```diff
@@ -40,9 +40,8 @@ func (suite *AccessSuite) TearDownTest() {
 }
 
 func (suite *AccessSuite) SetupTest() {
-	nodeConfigs := []testnet.NodeConfig{
-		testnet.NewNodeConfig(flow.RoleAccess),
-	}
+	acsConfig := testnet.NewNodeConfig(flow.RoleAccess)
```

Comment:
Minor thing, but the standard shorthands we use for the different roles is AN/LN/SN/VN/EN